flow/project.py

```diff
@@ -1683,16 +1970,16 @@ def _generate_submit_script(self, _id, operations, template, show_template_help,
             self._show_template_help_and_exit(template_environment, context)
         return template.render(** context)
 
-    def submit_operations(self, operations, _id=None, env=None, parallel=False, flags=None,
+    def submit_operations(self, groups, _id=None, env=None, parallel=False, flags=None,
```

Comment:
There is some API dissonance here now. The function is called `submit_operations`, but the first argument is named `groups`. Any ideas on how we could resolve this?

Reply:
Since the objects passed are JobOperations, I just renamed the argument to operations again. That was a holdover when there was submit_groups and submit_operations.